BedrockServer.h

```diff
@@ -390,11 +390,23 @@ class BedrockServer : public SQLiteServer {
     // The maximum number of conflicts we'll accept before forwarding a command to the sync thread.
     atomic<int> _maxConflictRetries;
 
+    // This contains all of the command that the previous list points at. This allows us to keep only a single copy of
```

Comment:
"previous list" ? Which previous list? :/

Reply:
Lol, during merge conflict resolution, I reversed the order in which `_outstandingHTTPSCommands` and `_outstandingHTTPSRequests` appeared in this file. I'll just make it explicit which list it refers to.